conans/test/command/export_linter_test.py

```diff
@@ -125,3 +125,19 @@ def test_warning_as_errors(self):
         self._check_linter(client.user_io.out)
         self.assertIn("ERROR: Package recipe has linter errors. Please fix them",
                       client.user_io.out)
+
+    def export_deploy_test(self):
+
+        conanfile = """
+from conans import ConanFile
+class BaseConan(ConanFile):
+    name = "baselib"
+    version = "1.0"
+
+    def deploy(self):
+        self.copy_deps("*.dll")
+"""
+        client = TestClient()
+        client.save({CONANFILE: conanfile})
+        client.run("export . conan/stable")
+        self.assertNotIn("Instance of 'BaseConan' has no 'copy_deps' member", client.out)
```

Comment:
A bit fragile, if the error message changes. Look for more generic linter warnings.

Make sure to check the full warnings, you don't get a "callable" warning?

If so, maybe the solution is to add it to export_linter.py